vagrant/k8s/setup_cluster.sh

```diff
@@ -48,15 +48,28 @@ function GetContiv {
   fi
 }
 
+#GetContrib fetches contrib if not present
+
+function GetContrib {
+	pushd .
+	if [ -f $top_dir/contrib ]; then
+    		echo "netplugin-$contivVer.tar.bz2 found, not fetching."
+  	else
+		git clone https://github.com/jojimt/contrib -b contiv 
+  	fi
+	popd
+}
+
 # kubernetes version to use -- defaults to v1.2.3
 : ${k8sVer:=v1.2.3}
 
 # contiv version
-: ${contivVer:=v0.1-05-05-2016.18-38-56.UTC}
+: ${contivVer:=v0.1-09-08-2016.20-56-40.UTC}
```

Comment:
Got it. Thanks ! Usually after some time we might need to update this default version. We have been facing some issues in net_demo_installer script as it has some old hard coded version but you can set that env variable and change it to new version.

Anyways this looks good then !